src/hotspot/share/gc/shenandoah/shenandoahHeap.inline.hpp

```diff
@@ -647,4 +648,15 @@ inline ShenandoahMarkingContext* ShenandoahHeap::marking_context() const {
   return _marking_context;
 }
 
+inline void ShenandoahHeap::set_cit(KlassInfoTable* cit) {
+  assert((_cit == nullptr && cit != nullptr) ||
+    (_cit != nullptr && cit == nullptr), "Initialize once & clear once");
+  _cit = cit;
+}
+
+inline KlassInfoTable* ShenandoahHeap::get_cit() {
+  assert(_cit != nullptr, "KlassInfoTable for Shenandoah should be initialized");
```

Comment:
Change the message to: "KlassInfoTable is null"
(suggestion is to avoid "initialized" here).